app/controllers/questions_controller.rb

```diff
@@ -70,11 +183,32 @@ def log_time
   end
 
   def next
-    setup = Setup.take
-    week = setup.weeks.find_by_number(params[:week_number])
-    RecommenderSystem::Recommender.setup(current_user.id,week.id)
-    best = RecommenderSystem::HybridRecommender.new.get_best
-    los = LearningObject.find(best[0])
-    redirect_to action: "show", id: los.url_name
```

Comment:
Neviem, to je otazka na Veroniku a profesroku :), ci chceme ten odporucac takto odstavit.